CONTRIBUTING.md

```diff
@@ -45,18 +45,31 @@ Repo maintainers can assign you an issue or pull request by leaving a
 
 After your PR is ready to commit, please run following commands to check your code.
 
-```shell
-make verify
-make test
-```
+- verify your code 
+  ```shell
+  make verify
+  ```
+- run the unit test
+  ```shell
+  make test
+  ```
+- run the integration test
+  ```shell
+  make test-integration
+  ```
+- run the end-to-end test, prepare a kind cluster and run the following command
```

Comment:
I think we need to load the image to kind at first?

Reply:
yes, updated